### PR TITLE
tests/e2e: Use epc.Etcdctl

### PR DESCRIPTION
**Pull request by serathius**
cc @ahrtr @ptabor

### CODE REVIEW

File: tests/e2e/etcd_config_test.go [hunk 1]
```diff
@@ -411,10 +411,7 @@ func TestEtcdHealthyWithTinySnapshotCatchupEntries(t *testing.T) {
 	for i := 0; i < 10; i++ {
 		clientId := i
 		g.Go(func() error {
-			cc, err := e2e.NewEtcdctl(epc.Cfg.Client, epc.EndpointsGRPC())
```
Comment (jmhbnz) [inline tests/e2e/etcd_config_test.go:414]: Grpc e2e failing with data race after this removal. I'm not sure we can remove this instance of the old approach as the test is intended to simulate 10 different clients?

Comment (serathius) [reply tests/e2e/etcd_config_test.go:414]: `Etcdctl` is not a client, it a wrapper over a etcdctl binary. Not only each `Etcdctl` call is unique, but also method call is unique execution of binary

Comment (jmhbnz) [reply tests/e2e/etcd_config_test.go:414]: Ahh right I see the spawns in `etcdctl.go`, apologies.  Any ideas why this change would be creating the data race with that in mind?

Comment (serathius) [reply tests/e2e/etcd_config_test.go:414]: Simple, I removed  scoped `err` variable.

Comment (jmhbnz) [reply tests/e2e/etcd_config_test.go:414]: Nice and checks now green. Thanks for indulging the questions.

**Review by jmhbnz — APPROVED** [review]
LGTM - Nice to cut down on all the duplication.

**Review by ahrtr — APPROVED** [review]
LGTM

Thanks

**Pull request merged**